data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,288 @@ mission "Arachi Cartoonist 2"
+			`	This process repeats a few more times. Finally, those that remain after several rounds - less than a tenth of those that entered - are gathered in the center. They leave the ring, and you become unsure if the game has ended.`
+			`	You feel something tap the top of your head and turn around to see an Arach sitting behind you, "Confused, the human visitor is?" Their little box translates.`
+			`	You nod, and they briefly breakdown the competition during this break. "Numerous, the Kimek are, so more athletes, they have," they say. "Over a thousand, at first enter. Redone, the teams are, once in half, that number is cut."`
+			`	They further explain that once the number of Kimek in the ring reaches two hundred and fifty, the game doesn't stop until only one hundred remain. The hundred reenter the ring and are paired off for individual matches using miniature rings drawn on the floor. This time they are all wearing what looks like goggles, perhaps as protective gear.`
+			`	You find Pairri is set to fight close to where you're sitting, so you get a good view of her fights. While she is short even by Kimek standards, she stands her ground and manages to beat some slightly larger opponents, driving them out of the circular limit of the rink.`
+			`	Above the large arena, monitors display the fighters and show their score: fighters that lose a match don't leave, but instead rotate counter-clockwise, fighting others to try and climb back up to the winning spots.`
+			`	Eventually, after many of these rounds and rotational swap outs, Pairri and her next opponent wait as the other forty-nine pairs finish their own fights, then move to the center of the field.`
+			`	"Undefeated, both of them are," the Arach behind you says. "The floor victor, the winner of this final match will be."`
+			`	The whistle sounds again, and Pairri charges at her slightly larger opponent, pressuring them toward the edge. They redirect the force by going to their side, and the fight quickly turns into a battle of attrition near the edge of the ring. Her opponent repositions themself closer to the middle and tries to charge, but she closes the gap. They continue wrestling before Pairri presses the other Kimek towards the border again. Suddenly, her opponent manages to grip her with their frontal legs, and by lifting their own body with the other four, manages to push her off the ground and into the air.`
+			`	Both fall outside the ring, but Pairri hits the ground first. You hear the whistle once again, and as they get up and approach each other for what must be the Kimek equivalent of a handshake, it becomes clear that Pairri has lost.`
+			`	Heliarch agents make sure everyone leaving the floor proceeds in an orderly fashion, and you take the elevators back down, returning to the entrance. You see a second train there, and the Arach that talked to you explains that it leads to where the cooperative game will take place.`
```

Comment:
I'd add an extra sentence at the beginning of this, simply as a transition between the event and leaving.
```suggestion
			`	With the event finally over, the audience begins to leave. Heliarch agents make sure everyone leaving the floor proceeds in an orderly fashion, and you take the elevators back down, returning to the entrance. You see a second train there, and the Arach that talked to you explains that it leads to where the cooperative game will take place.`
```